Code_of_Conduct.md

```diff
@@ -0,0 +1,73 @@
+# Code of Conduct
+
+## Our Pledge
+
+We, as members, contributors, and leaders, commit to creating a respectful and inclusive environment for everyone in our community. We value diversity and will not tolerate harassment based on factors such as age, body size, visible or invisible disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, or sexual identity and orientation. Our shared mission should reflect a culture of mutual respect and inclusivity, less than will not be tolerated.
+
+## Our Standards 
+
+### Acceptable Behavior Includes:
+
+* Demonstrate empathy, kindness, and respect towards others.
+* Respect differing opinions, perspectives, and experiences.
+* Give and receive constructive criticism respectfully.
+* Take responsibility for our actions and apologize when necessary.
+* Prioritizing the collective well-being of the community over individual interests.
+
+### Unacceptable Behavior Includes:
+
+* Using sexualized language or imagery, or making any form of sexual advances and/or harassment.
+* Engaging in trolling, derogatory comments, or launching personal or political attacks.
+* Harassing others in public or private settings.
+* Disclosing others' private information, such as physical or email addresses, without explicit consent (doxing).
+* Conduct that would be considered inappropriate in a professional context.
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of acceptable behavior and will take appropriate and fair corrective action in response to any behavior that they deem inappropriate, threatening, offensive, or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with this Code of Conduct and will communicate reasons for moderation decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces and also applies when an individual is officially representing the community in public spaces. Examples of representing our community include using an official e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## Enforcement
+
+If you or someone you know has experienced this type of behavior, please don't hesitate to reach out to the maintainer of the project directly at pauljod@msudenver.edu. Let's handle issues maturely and constructively. 
```

Comment:
The email address to use for reporting violations for this project is 8ibu5tidm20t@opayq.com